### DISCUSSION TITLE
Support for either/or validation?

### DISCUSSION THREAD

**Discussion by slimm609** [sse-secure-systems/connaisseur#655] [Q&A]
I couldn't find it in the documentation.  Does connaisseur support validating a single image with an either or type of policy to support migration from one signing technology to another?  Like moving from notary to cosign but being able to validate that it is signed by at least one of them to support a phased migration

**Comment by xopham** [comment]
@slimm609 you are correct: validating against two potential validators is currently not supported. You may get away with a more fancy image policy selecting by tag which would however cause higher maintenance efforts for the transition period. It may be possible to add something like this, but not yet sure how to to this in a natural way...
One might allow both validators in case the `pattern` is identical (a case in which Connaisseur would not know what to do anyways). However, that would be somewhat implicit as to which operator `AND` or `OR` would be applied and intuitively one may presume `AND`.
